ui/components/src/LineChart/LineChart.tsx

```diff
@@ -213,7 +215,14 @@ export function LineChart({
       onMouseEnter={handleOnMouseEnter}
     >
       {showTooltip === true && (
-        <Tooltip chartRef={chartRef} chartData={data} wrapLabels={true} pinTooltip={pinTooltip} unit={unit}></Tooltip>
+        <Tooltip
+          chartRef={chartRef}
+          chartData={data}
+          wrapLabels={true}
+          pinTooltip={pinTooltip}
+          unit={unit}
+          timeZone={timeZone}
```

Comment:
Suggestion (feel free to ignore): looks like maybe you could avoid passing this value as a prop through a couple levels of components if `TooltipContent` also called `useTimeZone` instead.

Reply:
I will explore this option